src/internal/item-container/index.tsx

```diff
@@ -174,6 +174,18 @@ export const ItemContainer = forwardRef(ItemContainerComponent);
 // isn't set in stone.
 export const CLICK_DRAG_THRESHOLD = 3;
 
+// We use singleton helper to avoid creating event listeners per each board item.
```

Comment:
Great improvement! It did not make sense having the global event listeners attached on every board item!